presto-benchto-benchmarks/src/test/java/io/prestosql/sql/planner/TestTpcdsCostBasedPlan.java

```diff
@@ -43,24 +44,24 @@
      * large amount of data.
      */
 
-    public TestTpcdsCostBasedPlan()
+    @Override
+    protected Optional<LocalQueryRunner> createLocalQueryRunner()
```

Comment:
I don't like this method returns `Optional`. What is the behaviour when it returns `Optional.empty()`? Maybe it should just have a default implementation instead?

Reply:
I see. adding default implementation in the `BasePlanTest::createLocalQueryRunner`.